slurm_config.yml

```
@@ -0,0 +1,9 @@
+logs_dir: slurm_logs
+parameters:
+  timeout_min: 70
+  slurm_partition: normal
+  slurm_gpus_per_task: 0
+  slurm_additional_parameters:
+    hint: nomultithread
+  slurm_exclusive: True
+  slurm_cpus_per_task: 40
```

If we use `nomultithread` (to disable hyper-threading / SMT), we should target 20 cpus per task to be able to use the base nodes on the margaret cluster.

```suggestion
  slurm_cpus_per_task: 20
```